### DISCUSSION TITLE
artemis屏蔽了系统三指触屏快捷弹出系统输入法软键盘吗？

### DISCUSSION THREAD

**Discussion by MuJian669** [ClassicOldSong/moonlight-android#59] [Ideas]
artemis发现三点触控弹出软键盘不起作用，这个是因为做虚拟按键给屏蔽了吗？虽然在返回菜单里有，但是还是希望恢复支持，给个选项也可以。

**Comment by ClassicOldSong** [comment]
鼠标模式选择触摸板的时候三指呼出键盘可用

但如果手机系统也有三指快捷操作的话触发可能会比较困难或者根本无法触发

**Reply by MuJian669** [reply]
> 鼠标模式选择触摸板的时候三指呼出键盘可用
> 
> 但如果手机系统也有三指快捷操作的话触发可能会比较困难或者根本无法触发

感谢，改成触摸板模式可以触发了
